source/Generic/PlayState/packages.config

```diff
@@ -1,5 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="PlayniteSDK" version="6.2.2" targetFramework="net462" />
-  <package id="System.Management" version="5.0.0" targetFramework="net462" />
-</packages>
```

Comment:
Why was this removed?

Reply:
Was automatically removed by Visual Studio when changing to package reference: https://docs.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference

This change was necessary because it didn't compile after adding the notifications dependencies without doing so.

Reply:
Okay, seems that it wasn't necessary. I reverted this change and added notifications on packages.config file.